src/dune_engine/rule.mli

```diff
@@ -50,6 +50,8 @@ module Id : sig
 
   val compare : t -> t -> Ordering.t
 
+  val dummy : t
```

Comment:
How is this used?

Reply:
It isn't. It is a leftover I forgot to remove. Thanks for pointing that out. I removed it